edge-modules/iotedge-diagnostics-dotnet/docker/linux/arm64v8/base/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG base_tag=3.1.5-bionic-arm64v8
```

Comment:
you don't need to this docker file as we only need 1 copy of docker file for module-base docker image.

Reply:
its modeled after temp sensor, which does that

Reply:
tempSensor is the only module containing azureiotedge-module-base image docker file